### ISSUE TITLE
Helm Chart - Issue with minio jobs (create user and the 2nd one) on a fresh PV

### ISSUE THREAD

**Issue by f-lalonde** [grafana/loki#11873]
**Describe the bug**
In short : the jobs seem to only activate once everything is ready in the helm release, however the `loki-write` pods never become ready because they can't access the bucket.
 
When specifying "minio.enabled: true" in the values of the loki helmchart, there seem to be a problem with the order of creation. loki-write never becomes ready, because the bucket it is trying to write to doesn't exists in `loki-minio`. However, once the helmrelease is ready, then the jobs seem to be activated.

**To Reproduce**
1. (If previously installed) Delete the PVs `data-loki-write`, or make sure to purge `loki-minio` of all buckets
2. (If previously installed) Uninstall the helm chart.
3. Install the helm chart (again), making sure to have specified "minio.enabled = true" with your preferred method (I do flux + kustomize)
4. The helm release is stuck with loki-write in a never ready state, and all logs contain errors too (for the same reason, but they do become ready)
5. Workaround :  Create a port-forward to the `loki-minio-0` pod on port 9001. Grab the `loki-minio` secrets too, and log in at `localhost:9001`
6. manually create the buckets : 
    - chunks
    - ruler
    - admin

(Now, if you are monitoring your pods, once everyone is marked as ready, you'll see two pods for the "make-bucket" job(s) created.

**Expected behavior**
Either the `loki-write` pods should become 'ready' even though they can't write in the bucket, or the jobs associated to `loki-minio` should be launched right after the `loki-minio`'s StatefulSet is marked as ready

**Environment:**
 - Infrastructure: Kubernetes 
 - Deployment tool: helm

**Screenshots, Promtail config, or terminal output**
Let me know if you need any, I haven't captured them and feel they won't add much.

**Comment by tgmuender** [comment]
This seems like a duplicate of https://github.com/grafana/loki/issues/9634

**Comment by JStickler** [comment]
Closing as Stale
As of March 16, 2026, the Loki Helm Chart has been forked to the new [Grafana-Community/helm-charts repo](https://github.com/grafana-community/helm-charts) and is now being maintained by the Grafana Community and Grafana Champions. If this issue affects Grafana Enterprise Logs (GEL), please leave a comment here.  Otherwise, if this is still an issue, please reopen your issue here [here](https://github.com/grafana-community/helm-charts/issues).
